packages/daemon/src/config.ts

```diff
@@ -48,13 +50,18 @@ export const DB_USER = process.env.DB_USER;
 export const DB_PASS = process.env.DB_PASS;
 export const DB_PORT = parseInt(process.env.DB_PORT ?? '3306', 10);
 
+// Other
+export const USE_SSL = process.env.USE_SSL;
+
 
 export default () => ({
   SERVICE_NAME,
   CONSOLE_LEVEL,
   TX_CACHE_SIZE,
   BLOCK_REWARD_LOCK,
```

Comment:
Should we remove this, and its definition above?

Reply:
Yes we should, done in f4c2fd24fae2068e1961b52012015fb90f27b8a2

Reply:
Thanks!